.eslintrc.js

```diff
@@ -0,0 +1,80 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  parserOptions: { ecmaVersion: 2018, sourceType: 'module', project: './tsconfig.json' },
+  plugins: [
+    '@typescript-eslint/eslint-plugin',
+    'prettier',
+    'unicorn',
+    'ava',
+    'import',
+  ],
+  extends: [
+    'eslint:recommended',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:unicorn/recommended',
+    'plugin:import/typescript',
+    'plugin:ava/recommended',
+    'xo',
+    'xo-typescript',
+  ],
```

Comment:
Interesting, why don't `xo` and `xo-typescript` have to be listed in the `plugins` option? And why they don't need the `plugin:` prefix here?

Reply:
`extends` is for configs - basically sets of rules; and `plugins` is for things that do pre-/post-processing on the code. See https://eslint.org/docs/developer-guide/working-with-plugins#configs-in-plugins

For the `plugin:` prefix: plugins can include optional configs, and the syntax for using them is `plugin:pluginName/configName`. As for `xo` and `xo-typescript` - those are the shorthand way of referring to the npm packages `eslint-config-xo` and `eslint-config-xo-typescript`.